File: 2020/01/2.py
inp = open("input", 'r')

ns = set()
for n in inp:
    ns.add(int(n))

for n in ns:
    for k in ns:
        if 2020 - n - k in ns:
            print(n * k * (2020 - n - k))
            exit(0)

exit(1)

File: 2022/01/2.jl
open("input") do i
    cals = Int[]
    acc = 0
    for line in readlines(i)
        if line == ""
            push!(cals, acc)
            acc = 0
        else
            acc += parse(Int, line)
        end
    end

    println(sum(sort(cals)[end-2:end]))
end
